llvm/runtimes/CMakeLists.txt

```
@@ -17,7 +17,11 @@ foreach(proj ${LLVM_ENABLE_RUNTIMES})
 endforeach()
 
 function(get_compiler_rt_path path)
-  foreach(entry ${runtimes})
+  set(all_runtimes ${runtimes})
+  foreach(name ${LLVM_RUNTIME_TARGETS})
+    list(APPEND all_runtimes RUNTIMES_${name}_LLVM_ENABLE_RUNTIMES)
```

I think you need to evaluate the variable otherwise you're just going to add `RUNTIMES_${name}_LLVM_ENABLE_RUNTIMES` as an element to the list.
```suggestion
    list(APPEND all_runtimes ${RUNTIMES_${name}_LLVM_ENABLE_RUNTIMES})
```